pkg/sshfs/nodeserver.go

```diff
@@ -128,6 +129,9 @@ func validateVolumeContext(req *csi.NodePublishVolumeRequest) error {
 	if _, ok := req.GetVolumeContext()["server"]; !ok {
 		return status.Errorf(codes.InvalidArgument, "missing volume context value: server")
 	}
+	if _, ok := req.GetVolumeContext()["port"]; !ok {
```

Comment:
I don't want to force the user to specify the port in any cases. I think it would be better to use port 22 as default if no port is specified. If it is specified, the specified port should be used.